modules/kn-trigger-update.adoc

```diff
@@ -1,43 +1,42 @@
 // Module included in the following assemblies:
 //
-// * serverless/event_workflows/serverless-kn-trigger.adoc
+// * /serverless/event_workflows/serverless-using-brokers.adoc
 
 [id="kn-trigger-update_{context}"]
 = Updating a trigger using `kn`
 
 You can use the `kn trigger update` command with certain flags to update attributes for a trigger.
 
-.Example
-
-. To update a trigger, enter the following command:
-+
-
+.Example command
```

Comment:
I'm not going to push too hard on this now since the file was already like this and I know your GA is coming up, but this looks to be a procedure module, so it should follow the whole procedure module format (with `.Procedure`, etc.). Maybe that's something you could take a look at in a future PR?